pyvrp/cpp/TimeWindowSegment.h

```diff
@@ -8,12 +8,13 @@ class TimeWindowSegment
 {
     using TWS = TimeWindowSegment;
 
-    int idxFirst = 0;       // Index of the first client in the segment
-    int idxLast = 0;        // Index of the last client in the segment
-    Duration duration = 0;  // Total duration, incl. waiting and servicing
-    Duration timeWarp = 0;  // Cumulative time warp
-    Duration twEarly = 0;   // Earliest visit moment of first client
-    Duration twLate = 0;    // Latest visit moment of last client
+    int idxFirst = 0;          // Index of the first client in the segment
+    int idxLast = 0;           // Index of the last client in the segment
+    Duration duration = 0;     // Total duration, incl. waiting and servicing
+    Duration timeWarp = 0;     // Cumulative time warp
+    Duration twEarly = 0;      // Earliest visit moment of first client
+    Duration twLate = 0;       // Latest visit moment of first client
```

Comment:
Changed "last" to "first" here (per [Cattaruzza et al. (2016)](https://pubsonline.informs.org/doi/abs/10.1287/trsc.2015.0608)).